src/App.css

```diff
@@ -39,52 +68,199 @@
 
 .bookContainer {
   display: flex;
+  flex-direction: row;
   justify-content: space-between;
-  padding: 4rem;
-  border: 1px solid black;
-  gap: 1rem;
-  align-items: center;
+  background-color: #fff;
+  margin: 1% 5%;
+  padding: 1% 10% 1% 2%;
+  font-family: 'Roboto Slab', serif;
+  border: solid 1px #e8e8e8;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
 }
 
-.buttonsList {
+.bookTitle {
   display: flex;
+  flex-direction: column;
   gap: 1rem;
 }
 
-.bookTitle {
+.category {
+  color: #121212;
+  opacity: 0.5;
+  font-weight: bold;
+}
+
+.title {
+  color: #121212;
+  font-size: larger;
+  font-weight: bold;
+}
+
+.author {
+  color: #4386bf;
+}
+
+.buttonsList {
   display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-  width: 20rem;
+  gap: 1rem;
 }
 
 .buttonsList li {
   list-style: none;
 }
 
-.bookChart {
+.bookBtn {
+  outline: none;
+  border: none;
+  background-color: #fff;
+  color: #4386bf;
+  cursor: pointer;
+  padding-left: 0%;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+}
+
+.progress {
   display: flex;
-  gap: 0.5rem;
-  width: 10rem;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  min-width: 35%;
 }
 
-.bookChapter {
+.chart {
   display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-  width: 20rem;
+  flex-direction: row;
+  align-items: center;
 }
 
-.addBook {
-  margin-top: 1rem;
+.ProgressBar {
+  margin-right: 10%;
+}
+
+.circle-wrap {
+  width: 75px;
+  height: 75px;
+  background: #e8e8e8;
+  border-radius: 50%;
+  border: 1px solid #cdcbd0;
+}
+
+.mask .fill {
+  clip: rect(0, 37.5px, 75px, 0);
+  background-color: #227ded;
+}
+
+.mask.full,
+.circle .fill {
+  animation: fill ease-in-out 3s;
+  transform: rotate(135deg);
+}
+
+.circle-wrap .circle .mask,
+.circle-wrap .circle .fill {
+  width: 75px;
+  height: 75px;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.circle-wrap .circle .mask {
```

Comment:
- As suggested by the previous reviewer, 

Review Comment:

Great job on your progress so far! 👍 I noticed that the circle representing the progress is missing from your design. To align it with the Zeplin design, please follow these steps:

1. Element Creation: Create the circle element in your HTML file using appropriate markup, such as a `<div>` or `<span>`. Assign it a unique class or ID for targeting in your CSS.

2. Styling: Apply the necessary styles to the circle element in your CSS file to achieve the desired appearance specified in the Zeplin design. This may include defining the size, background color, border, or any other relevant properties.

3. Positioning: Position the circle element appropriately within your design to match the placement indicated in the Zeplin design. This might involve using CSS positioning properties like `absolute` or `relative` and adjusting margins or paddings.

By adding the circle element and applying the designated styles, you will bring your design in line with the Zeplin design. If you have any questions or need further guidance, please don't hesitate to ask. Keep up the excellent work! 💪